kuma/users/jinja2/socialaccount/includes/change_email_address.html

```diff
@@ -1,4 +1,5 @@
 <div id="change-email-container" class="hidden">
     <label for="{{ form.email.id_for_label }}" class="visually-hidden">{{ default_email }}</label>
     {{ form.email }}
+    <div id="{{ form.email.id_for_label }}-error" class="error hidden">{{ _('The email field is required.') }}</div>
```

Comment:
I don't understand why this is needed. First of all, doesn't the `{{ form.email }}` produce something like `<input name="email" required ...>` so that should take care of protection against omission. 
Second, if the user did type something that gets sent (e.g. `''` or `'usernamethatsalreadytaken'`) the `{{ form.email }}` 's own error handling should be able to display that.

Reply:
You are correct, and I was considering removing the email portion of the validation. Perhaps I was trying to be consistent or had an edge case in mind. That must have been a super sharp edge though. Let me think through this again and remove it if it serves no real purpose.